geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/DebugCommand.java

```diff
@@ -19,17 +19,16 @@
 import org.springframework.shell.core.annotation.CliOption;
 
 import org.apache.geode.management.cli.CliMetaData;
-import org.apache.geode.management.cli.Result;
+import org.apache.geode.management.cli.GfshCommand;
 import org.apache.geode.management.internal.cli.i18n.CliStrings;
-import org.apache.geode.management.internal.cli.result.ErrorResultData;
-import org.apache.geode.management.internal.cli.result.ResultBuilder;
+import org.apache.geode.management.internal.cli.result.model.ResultModel;
 import org.apache.geode.management.internal.cli.shell.Gfsh;
 
-public class DebugCommand extends InternalGfshCommand {
+public class DebugCommand extends GfshCommand {
```

Comment:
We should extend OfflineGfshCommand for all offline commands. It has getGfsh() methods to get the gfsh isntance.

Reply:
Right. Thanks!